src/RealtimeServer/common/models/project-rights.ts

```
@@ -27,9 +30,10 @@ export class ProjectRights {
   }
 
   hasRight(project: Project, userId: string, projectDomain: string, operation: Operation, data?: OwnedData): boolean {
-    const rights = (this.rights.get(project.userRoles[userId]) || []).concat(
-      (project.userPermissions || {})[userId] || []
-    );
+    const userRole: string = project.userRoles[userId];
+    const rights = (this.rights.get(userRole) || [])
+      .concat((project.userPermissions || {})[userId] || [])
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)